CHANGELOG.md

```diff
@@ -1,3 +1,24 @@
+## v0.3.0 [2022-07-20]
+
+_What's new?_
+
+- New table added
+  - [steampipecloud_organization_workspace_member](https://hub.steampipe.io/plugins/turbot/steampipecloud/tables/steampipecloud_organization_workspace_member) ([#19](https://github.com/turbot/steampipe-plugin-steampipecloud/pull/19))
+
+_Enhancements_
+
+- [steampipecloud_organization_workspace_mod](https://hub.steampipe.io/plugins/turbot/steampipecloud/tables/steampipecloud_workspace_mod)
+  - Renamed column `mod_constraint` to `constraint`
```

Comment:
These are breaking changes, what's the reason for making these?

Reply:
So is the response data different then? The transform looks the same, so do we get the correct value back for that column now? For instance, if the response field is now called `Constraint` and was previously `ModConstraint`, our transform wouldn't work anymore.